### PR TITLE
[MINOR] QMS: fix collaborator editor loading

### PR DESCRIPTION
**Pull request by annano**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: packages/text-editor/src/components/CollaboratorEditor.svelte [hunk 1]
```diff
@@ -101,6 +101,8 @@
     provider?.on('synced', () => {
       loading = false
     })
+  } else {
+    loading = false
```
Comment (aonnikov) [inline packages/text-editor/src/components/CollaboratorEditor.svelte:105]: do you use context provider?

**Review by aonnikov — APPROVED** [review]

File: packages/text-editor/src/components/CollaboratorEditor.svelte (hunk 1 above)
Comment (sol-0) [reply packages/text-editor/src/components/CollaboratorEditor.svelte:105]: Yes, it's used through `Collaboration.svelte`

**Review by sol-0 — COMMENTED** [review]
@aonnikov @annano  Do we also need to wait for `synced` from `contextProvider`?

File: packages/text-editor/src/components/CollaboratorEditor.svelte (hunk 1 above)
Comment (aonnikov) [reply packages/text-editor/src/components/CollaboratorEditor.svelte:105]: Ah, ok, I overlooked this somehow. We need to investigate how to wait  for loading in this case then.

**Pull request merged**